File: admin/src/main/java/bigburguer/admin/model/PratoModel.java
package bigburguer.admin.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;

@Entity(name = "prato")
public class PratoModel {
    @Id
    private Integer id = 0;
    @Column(nullable = false, length = 100)
    private String nome;
    @Column(nullable = false, length = 10)
    private Double preco;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public Double getPreco() {
        return preco;
    }

    public void setPreco(Double preco) {
        this.preco = preco;
    }


}

File: garcom/lib/models/prato_model.dart
class Prato {
  int? id;
  double? preco;
  String? nome;
  Prato(
    this.id,
    this.preco,
    this.nome,
  );
  Prato.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    preco = json['preco'];
    nome = json['nome'];
  }
  Map<String, dynamic> toJson() {
    return <String, dynamic>{
      "id": id,
      "preco": preco,
      "nome": nome,
    };
  }
}
